scripts/how_to/adding_new_ida_to_toggle_report.rst

```diff
@@ -0,0 +1,135 @@
+=====================================
+Enabling Toggles Report for a new IDA
+=====================================
+
+The generation of toggles report for an IDA has been automated. However, for now, the automation has only been used on lms. This document lists what needs to be done to automate toggle report generation for other IDAs.
+
+.. contents:: Steps
+
+Gathering Data
+==============
+
+Annotation Data
+---------------
+The toggles are should be annotated in code following edX code annotation:`writing annotations`_. The `code_annotations`_ tool is used to collect annotations into a report.
+
+The steps to collect annotations is automated through a jenkins job
+  - `groovy job specification`_ in "generate-code-annotation-report"
+  - `Jenkins Job folder`_
+  - once the job is done, its data is pushed to s3 bucket: `script to push data to s3 bucket`_
+
+
+.. _writing anntations: https://code-annotations.readthedocs.io/en/latest/writing_annotations.html
+.. _code_annotations: https://www.github.com/edx/code-annotations
+
+steps
+~~~~~
+* add a `generate-feature-toggle-annotation-report.sh`_ to repository, use linked file as example
```

Comment:
Does each repo need its own annotation report generation script? What parts will differ by IDA?

Reply:
Not much from what I can tell/this is a open question.